### ISSUE TITLE
Multiple hypervisor support

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#95]
Originally reported of Google Code with ID 39.

```
Opening bug for tracking multiple hypervisor support.
```
Originally added on 2008-07-21 15:21:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2008-08-05 10:11:04 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This was fixed in 2.0.
```
Originally added on 2009-08-05 08:50:06 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-22 08:57:07 +0000 UTC.

Changed State: Released
